frontend/model/contracts/Contract.js

```diff
@@ -3,9 +3,17 @@
 import sbp from '~/shared/sbp.js'
 import { GIMessage } from '~/shared/GIMessage.js'
 
+// this must not be exported, but instead accessed through 'actionWhitelisted'
+const whitelistedSelectors = {}
+
+// can be useful for extracting the name of the contract or action, for example:
+// const contractName = CONTRACT_REGEX.exec(selector)[1]
+export const CONTRACT_REGEX = /^[\w.]+\/([^/]+)\/(?:([^/]+)\/)?process$/
```

Comment:
I think something should be exported when there's a real need for it. If there isn't one yet, let it be as a simple variable. Keep it simple.

Reply:
Well there is, it's used [here in `state.js`](https://github.com/okTurtles/group-income-simple/pull/693/files#diff-1ae533b411af6643b2d8345264dd5499R395)